webui/src/pages/user/user-settings-profile.tsx

```diff
@@ -58,6 +66,13 @@ class UserSettingsProfileComponent extends React.Component<UserSettingsProfileCo
                     <Typography variant='h5' gutterBottom>Profile</Typography>
                     <Typography variant='body1'>Login name: {this.props.user.loginName}</Typography>
                     <Typography variant='body1'>Full name: {this.props.user.fullName}</Typography>
+                    <div className={this.props.classes.theme} aria-hidden="true">
+                        <Typography variant="body1">Theme: </Typography>
+                        <ThemeSwitcher
+                            darkMode={this.props.darkMode}
+                            setDarkMode={this.props.setDarkMode}
+                        />
```

Comment:
I would prefer a normal switch with label "Dark theme". Later we might change this to a three-value switch (autodetect, light, dark).